contracts/VaultV3.vy

```diff
@@ -272,16 +271,21 @@ PERMIT_TYPE_HASH: constant(bytes32) = keccak256("Permit(address owner,address sp
 
 # Constructor
 @external
-def __init__(
-    asset: ERC20, 
+def __init__():
+    # Set `asset` so it cannot be re-initialized.
+    self.asset = self
```

Comment:
does this need to call initialize?

Reply:
Assuming the first one deployed is just a dummy version to use for cloning.

So just want to make sure it can't be initialized ever and only used as the original